modules/channel-web/src/backend/index.ts

```diff
@@ -20,10 +22,16 @@ const onModuleUnmount = async (bp: typeof sdk) => {
   bp.http.deleteRouterForBot('channel-web')
 }
 
+const onMigrationRequest = async (bp: typeof sdk, dryRun?: boolean) => {
```

Comment:
why do we need this method?  It would be better IMO if Botpress just automatically executed all migrations in the `/migrations` folder, by alphabetical order, passing it an instance of `bp`

Reply:
Since some migrations may require access to other stuff (ex: storage or business logic) to save changes to files, I found it easier this way, but I'm definitely open for a more efficient way to handle that

Reply:
@slvnperron  While having a `/migrations` folder is a standard approach, how would you handle (potentially 3rd party) module migrations ? they would be copied just like we do with actions, hooks, etc ?